reports/tests.py

```diff
@@ -67,35 +67,6 @@ def setUp(self):
             github_handle='organization_github_handle'
         )
 
-    @ddt.data(
-        None,  # Anonymous User
-        'user1',  # Normal User
-        'user2',  # Staff User
-        'user4',  # Sandbox user
-    )
-    def test_report_view_not_superuser(self, username):
```

Comment:
This should not be removed, the login view should still exist.

Reply:
I have unremoved it but since it takes reverse resolver for registration URL, I need help here to make it not fail